app/packages/core/src/components/ColorModal/controls/ColorAttribute.tsx

```diff
@@ -58,24 +58,27 @@ const ColorAttribute: React.FC<Prop> = ({ style }) => {
 
   const setColorScheme = fos.useSetSessionColorScheme();
   const activeField = useRecoilValue(activeColorField);
-  const { colorPool, fields } = useRecoilValue(fos.colorScheme);
-  const index = fields.findIndex((s) => s.path == activeField.path);
+  const currentColorScheme = useRecoilValue(fos.colorScheme);
+  const index = currentColorScheme.fields.findIndex(
+    (s) => s.path == activeField.path
+  );
 
   const options = subfields.map((field) => ({
     value: field.path?.split(".").slice(-1)[0],
```

Comment:
nit: `field.path?.split(".").slice(-1)[0]` is used here and on line 72. Maybe the map function could hold it as a constant?